src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -67,377 +65,436 @@
-					script = TestMode.lastTestFile;
-					if (script == null) {
-						Skript.error(sender, "No test script has been run yet!");
-						return true;
-					}
-				} else {
-					script = TestMode.TEST_DIR.resolve(
-							Arrays.stream(args).skip(1).collect(Collectors.joining(" ")) + ".sk").toFile();
-					TestMode.lastTestFile = script;
-				}
-				assert script != null;
-				if (!script.exists()) {
-					Skript.error(sender, "Test script doesn't exist!");
-					return true;
-				}
-				
-				ScriptLoader.loadScripts(ScriptLoader.loadStructure(script)); // Load test
-				Bukkit.getPluginManager().callEvent(new SkriptTestEvent()); // Run it
-				Skript.disableScripts(); // Clean state for next test
-				
-				// Get results and show them
-				String[] lines = TestTracker.collectResults().createReport().split("\n");
-				for (String line : lines) {
-					assert line != null;
-					Skript.info(sender, line);
-				}
-			}
-		} catch (final Exception e) {
-			Skript.exception(e, "Exception occurred in Skript's main command", "Used command: /" + label + " " + StringUtils.join(args, " "));
-		} finally {
-			r.stop();
-		}
-		return true;
-	}
-	
-	private final static ArgsMessage m_invalid_script = new ArgsMessage(NODE + ".invalid script");
-	private final static ArgsMessage m_invalid_folder = new ArgsMessage(NODE + ".invalid folder");
-	
-	@Nullable
-	private static File getScriptFromArgs(final CommandSender sender, final String[] args, final int start) {
-		String script = StringUtils.join(args, " ", start, args.length);
-		File f = getScriptFromName(script);
-		if (f == null){
-			Skript.error(sender, (script.endsWith("/") || script.endsWith("\\") ? m_invalid_folder : m_invalid_script).toString(script));
-			return null;
-		}
-		return f;
-	}
-	
-	@Nullable
-	public static File getScriptFromName(String script){
-		final boolean isFolder = script.endsWith("/") || script.endsWith("\\");
-		if (isFolder) {
-			script = script.replace('/', File.separatorChar).replace('\\', File.separatorChar);
-		} else if (!StringUtils.endsWithIgnoreCase(script, ".sk")) {
-			int dot = script.lastIndexOf('.');
-			if (dot > 0 && !script.substring(dot+1).equals("")) {
-				return null;
-			}
-			script = script + ".sk";
-		}
-		if (script.startsWith("-"))
-			script = script.substring(1);
-		File f = new File(Skript.getInstance().getDataFolder(), Skript.SCRIPTSFOLDER + File.separator + script);
-		if (!f.exists()) {
-			f = new File(f.getParentFile(), "-" + f.getName());
-			if (!f.exists()) {
-				return null;
-			}
-		}
-		return f;
-	}
-	
-	private static Collection<File> toggleScripts(final File folder, final boolean enable) throws IOException {
-		return FileUtils.renameAll(folder, new Converter<String, String>() {
-			@Override
-			@Nullable
-			public String convert(final String name) {
-				if (StringUtils.endsWithIgnoreCase(name, ".sk") && name.startsWith("-") == enable)
-					return enable ? name.substring(1) : "-" + name;
-				return null;
-			}
-		});
-	}

```

Comment:
All of this

Reply:
Isn't that a removed phrase?

Reply:
Wait. The problem is not line endings, the problem is you are replacing tabs with spaces... Your code **must** be tabbed as per the Skript coding conventions.